src/net_processing.cpp

```diff
@@ -2309,6 +2311,19 @@ void PeerManagerImpl::ProcessGetCFCheckPt(CNode& peer, CDataStream& vRecv)
     m_connman.PushMessage(&peer, std::move(msg));
 }
 
+void PeerManagerImpl::ProcessBlock(CNode& pfrom, const CChainParams& chainparams, const std::shared_ptr<const CBlock> pblock,
```

Comment:
Nit: you can turn `pblock` into a const reference argument (`const std::shared_ptr<const CBlock>& pblock`), which will avoid an atomic shared_ptr copy at runtime.

Reply:
Thanks. changed. If I may ask, why doesn't ProcessNewBlock() also do this?

Reply:
Probably just a typo